src/algebra/symmetrized.lean

```diff
@@ -5,6 +5,7 @@ Authors: Christopher Hoskin
 -/
 import algebra.module.basic
 import tactic.abel
+import algebra.jordan.basic
```

Comment:
Can you alphabetize these?

Reply:
Actually, `import algebra.jordan.basic` is enough.

Reply:
It's better to leave the other imports, assuming the file actually refers to `module` and uses `abel`.

Also, I suspect this will no logner work on master, where I've removed lots of import from `algebra.jordan.basic`

Reply:
I merged `master` and `leanproject build` ran without error. I've added `import algebra.module.basic` back in as `module` is referred to. `abel` isn't so I left it out.